test/unit/api/response/iiif/presentation-api/placeholder-canvas.test.js

```diff
@@ -0,0 +1,70 @@
+"use strict";
+
+const chai = require("chai");
+const {
+  getPlaceholderSizes,
+} = require("../../../../../../src/api/response/iiif/presentation-api/placeholder-canvas");
```

Comment:
We have the `requireSource` helper to make test code more readable.

```suggestion
} = requireSource("api/response/iiif/presentation-api/placeholder-canvas");
```